workspace-automation.md

```diff
@@ -0,0 +1,35 @@
+# Workspace Automation
+
+Over time, we've established various workspace automation to boost our productivity and get repetitive chores out of our way.
```

Comment:
maybe say 'boost our productivity and easily access information we are interested in knowing on a regular basis.'

That way its clear why it becomes repetitive.